lib/tasks/escalated_defects.rake

```diff
@@ -0,0 +1,14 @@
+# A collection of tasks used to trigger emails to the New Build Team about
+# defects in the system.
+
+namespace :notify do
+  task escalated_defects: :environment do
+    escalated_defects = Defect.open.flagged
+    EmailEscalatedDefects.new(defects: escalated_defects).call
```

Comment:
Given the naming of these classes, would it make more sense for then to not accept any arguments and then do the query in the classes themselves?

Reply:
I like to inject where possible into service classes, but I completely take your point.